### DISCUSSION TITLE
Set default version for runtimes

### DISCUSSION THREAD

**Discussion by chisNaN** [beginner-corp/begin-community#63] [Support] [answered]
Hello

Its a request i already opened during the summer in the older community platform. (and which remained pending)
As of now is there a possibility to set a default version for deno runtime
My backend application hosted on begin still work with the current version 1.3
but its a matter of time because my app is broken localy with 1.5.2 version of deno.

Would be glad to have some news about that.
thanks in advance

**Comment by brianleroux** [comment]
hey @chisNaN super sorry to leave you hanging: this is on me! upgrading us now and will update when thats tested and deployed to production

<img width="596" alt="Screen Shot 2020-11-14 at 9 41 15 AM" src="https://user-images.githubusercontent.com/990/99153474-8d567680-265d-11eb-829b-c19c9d1b90e1.png">

**Comment by brianleroux** [comment, marked as answer]
Deno v1.5.3 is live! Thanks again for your patience. We need to step up the pace to keep up with the Deno team! 🦖

**Comment by chisNaN** [comment]
Thanks for your reactivity @brianleroux 
But actually my app will not work anymore with the last version of deno. That would be too long to enumerate all the stuff broken.
My request is the following.
Is there a possibility to *set a default deno version* in begin?
I would like to stick with 1.3 for instance 

Thanks

**Reply by brianleroux** [reply]
Ouch sorry to hear thay. Ok, I've been intending to implement this functionality. Deno is still pretty frothy still slo pinning to latest seems safest generally until they get the config frozen especially. But I understand how it goes! 

I'll have a look at versioning and report back.

**Reply by chisNaN** [reply]
Thanks sir again. Ultimately I have a question.
When do you plan to exit free tier the begin platform?
Recently for instance I used a kubernetes cluster called kubesail which was free because they were still in development phase but now they turned all paid plans.
Thanks
ps: maybe i should open a new topic

**Reply by brianleroux** [reply]
Free tier will always be free and you will always be able to eject your app and deploy to your own AWS account with arc.codes and CloudFormation. We believe lock-in is not customer friendly and will never bait/switch you.

**Reply by chisNaN** [reply]
THanks BRYAN,

any update about setting a default deno version?
at first glance i thought deno was nice but in production its a nightmare
when deno version change or typescript even syntax error can arise
and also this idea of importing online crossdomain files is only viable if it is in your own server...
because otherwise themselves are prone to change and errors...

**Reply by brianleroux** [reply]
Hey sorry! Wall to wall meetings this past week but I do have some research here. Think its likely we can add this functionality 🔜 but can't commit to any dates yet! I agree Deno is a bit unstable at the moment. Good thing we have staging and production build in by default !

**Reply by chisNaN** [reply]
exactly, thanks BRIAN keep us informed about this next new deno feature
